modules/ROOT/pages/penetration-testing-policies.adoc

```diff
@@ -9,40 +9,78 @@ image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybr
 image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-platform-private-cloud-edition", title="Anypoint Platform PCE"]
 image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
 
-If you are deploying applications to xref:index.adoc[Runtime Manager] or APIs on xref:api-manager::index.adoc[API Manager] and your company's security policies require that you run penetration testing on them, there is a way for you to run these. Given that such tests can interfere with other tenants, penetration testing is only allowed on your own workers. It is not allowed for any other services in the Anypoint Platform.
+If your company security policies require that you perform vulnerability and penetration testing 
+against Anypoint Platform, request a Security Assessment before proceeding.
```

Comment:
Why capitalize "security assessment"?

Reply:
It's how it appears in all those Salesforce KB articles